### DISCUSSION TITLE
Possibility to download subtitles. Возможность скачать субтитры.

### DISCUSSION THREAD

**Discussion by Mastade** [ilyhalight/voice-over-translation#548] [Ideas]
Добавьте, пожалуйста , возможность скачать субтитры созданные Яндексом для видео на ютуб и т.д
Please add the ability to download subtitles created by Yandex for videos on YouTube, etc.

**Comment by NullVerdict** [comment]
> Добавьте, пожалуйста , возможность скачать субтитры созданные Яндексом для видео на ютуб и т.д Please add the ability to download subtitles created by Yandex for videos on YouTube, etc.

Уже есть, начиная с версии 1.5.0

**Reply by Mastade** [reply]
Спасибо, вроде понял как качать(справа возле "Настройки VOT" есть кнопка, но не всегда получается скачать) но у меня  в тексте скачанных субтитров очень много лишних слов, цифр, символов, это можно как исправить-удалить?

**Reply by NullVerdict** [reply]
#514

**Reply by Mastade** [reply]
Благодарю за ссылку на скрипт, но я полный чайник в них, буду пробовать онлайн сервисы для этого

**Comment by Mastade** [comment]
Ничего путного онлайн не нашел, всё на выходе нечитаемое получается, но всё-равно большое спасибо за желание помочь

**Comment by Lexx0109** [comment]
Добрый день.
Добавьте возможность менять размер шрифта субтитров, т.к. сейчас они очень маленькие и если смотришь через телевизор, то текст почти не читаемый.
